src/common/commonutils/UserUtils.c

```diff
@@ -3140,134 +3003,65 @@ int CheckUserAccountsNotFound(const char* names, char** reason, OsConfigLogHandl
     return status;
 }
 
-int RemoveUserAccounts(const char* names, bool removeHomeDirs, OsConfigLogHandle log)
+int RemoveUserAccounts(const char* names, OsConfigLogHandle log)
```

Comment:
>RemoveUserAccounts [](http://example.com/codeflow?start=4&length=18)

We need the warning trace for a home dir remaining after its owner user account is deleted in this function as well (actually, is even more important here than in the Check function). Do you think you could add?